x-pack/test/api_integration/apis/uptime/rest/snapshot.ts

```diff
@@ -6,47 +6,97 @@
 
 import { expectFixtureEql } from '../graphql/helpers/expect_fixture_eql';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { makeChecksWithStatus, getChecksDateRange } from '../graphql/helpers/make_checks';
 
 export default function({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
 
   describe('snapshot count', () => {
-    let dateRangeStart = '2019-01-28T17:40:08.078Z';
-    let dateRangeEnd = '2025-01-28T19:00:16.078Z';
-
-    it('will fetch the full set of snapshot counts', async () => {
-      const apiResponse = await supertest.get(
-        `/api/uptime/snapshot/count?dateRangeStart=${dateRangeStart}&dateRangeEnd=${dateRangeEnd}`
-      );
-      expectFixtureEql(apiResponse.body, 'snapshot');
-    });
+    const dateRangeStart = new Date().toISOString();
```

Comment:
Having looked at the internals and now seeing these declarations, I think we can eventually specify just one date (end time) and a number of seconds to subtract from it for our API calls.